src/components/CookieNotice/CookieNotice.scss

```diff
@@ -0,0 +1,59 @@
+@import "constants/style.scss";
+
+$cookie-consent-decline-button: #dadada;
+$cookie-consent-decline-text: #4d4d4d;
+$cookie-consent-button-text-size: 18px;
+
+.cookie-notice {
+  background: $color-light-gray;
+  color: $color-black;
+  position: fixed;
+  bottom: 0;
+  padding: $padding--small;
+}
+
+.cookie-notice__buttons {
+  display: flex;
+}
+
+.cookie-notice__button-accept {
+  background: $color-backlog-blue;
+  color: $color-white;
+  margin: 0 0 $margin--default $margin--default;
+  border: none;
+  padding: 5px 15px;
+  border-radius: 5px;
+  font-family: inherit;
+  font-size: $cookie-consent-button-text-size;
+}
+.cookie-notice__button-accept:hover {
```

Comment:
Is it really the preferred behaviour that a button is less visible on hover? 
I think I'd prefer something like `transform: scale(1.05)`

Reply:
Also I'd give `cookie-notice__button-accept`, `cookie-notice__button-decline` and  `cookie-notice__button-cookie-policy` an additional class `cookie-notice__button` and summarise all common css attributes there. 

For example all there buttons have the same hover effect, padding and border-radius.